comment/tests/test_migrations.py

```diff
@@ -37,4 +39,21 @@ def test_adjust_flagged_comments(self):
         self.assertTrue(comment_1.is_flagged)
 
 
-
+class CommentURLHashMigrationTest(BaseCommentMigrationTest):
+    migrate_from = '0007_auto_20200620_1259'
+    migrate_to = '0008_comment_urlhash'
+
+    def setUpBeforeMigration(self, apps):
+        comment_model = apps.get_model('comment', 'Comment')
+        self.comment_id = comment_model.objects.create(
+            content_type_id=self.ct_object.id,
+            object_id=1,
+            content='test urlhash',
+            user_id=self.user.id
```

Comment:
This should ideally be
```python

self.comment_id = comment_model.objects.create(
    content_object=self.ct_object,
    content='hi',
    user=self.user).id
```
But this was causing weird issues, probably because the model class here is `fake`. If you figure out and solve the issue, please inform me as well.

Reply:
I will look at it.

Reply:
I believe as you said because of using fake model. I cannot find another explaination.

Reply:
Also I have used `TransactionTestCase` instead of `TestCase` since it was causing issues with atomic requests.